<launch>
  <group ns="prosilica_view">
    <param name="window_name" type="str" value="Prosilica camera"/>
    <param name="autosize" type="bool" value="false"/>
  </group>
  <node name="prosilica_view" pkg="image_view" type="image_view" respawn="false" output="screen" cwd="node">
    <remap from="image" to="/prosilica/image_rect"/>
  </node>
</launch>
